nntrainer/src/model_loader.cpp

```diff
@@ -249,17 +249,27 @@ int ModelLoader::loadBackboneConfigExternal(dictionary *ini,
                                             const std::string &backbone_config,
                                             std::shared_ptr<Layer> &layer,
                                             const std::string &backbone_name) {
-#ifdef ENABLE_NNSTREAMER_BACKBONE
+  LayerType type = LayerType::LAYER_UNKNOWN;
+
+#if defined(ENABLE_NNSTREAMER_BACKBONE)
```

Comment:
Maybe later... how about we delegate && detect inside backbone layer and use only LayerType::LAYER_BACKBONE. I think we can say LAYER_BACKBONE has two implementations.